homeassistant/components/zwave_js/services.yaml

```diff
@@ -27,5 +27,40 @@ set_lock_usercode:
       description: Code to set.
       required: true
       example: 1234
```

Comment:
@marcelveldt looks like you accidentally wiped out the selector here 🙂

Reply:
hurray for my merging skills